### DISCUSSION TITLE
Gauge Chart Max Indicator

### DISCUSSION THREAD

**Discussion by digitalfr4nk** [Live-Charts/LiveCharts2#1195] [Q&A]
All,

Im trying to create a "max" indicator on a gauge chart (my gauge is used to display live data). 
I've achieved something similar with the angular gauge needle but I'd like to be just on the series, any ideas how to achieve this? I also thought of creating a second series/gauge and overlaying but I was hoping for something cleaner.

![image](https://github.com/beto-rodriguez/LiveCharts2/assets/79330257/7875b6f5-3291-42ba-8563-a0c6f2e56612)

Thanks,
Frank.

**Comment by beto-rodriguez** [comment]
Hello, can you please provide an image, so I get a better idea of your request?
